AxonDeepSeg/train_network.py

```diff
@@ -274,7 +274,7 @@ def train_model(
 
     ########################## Save the model after Training ###########
 
-    model.save(str(path_model) + "/model.hdf5")
+    model.save(str(path_model / "model.hdf5"))
```

Comment:
I presume the reason for this commit is being consistent in ADS code base on string concatenation. Is that right?

Reply:
Yup, exactly. I noticed it because that's the line that the test was failing on.